### DISCUSSION TITLE
Which coordinate system is used in Assimp?

### DISCUSSION THREAD

**Discussion by cj2000** [assimp/assimp#4815] [Q&A] [answered]
There are at least 2 diferent coordinate systems, but which one is used by Assimp left-handed or right-handed? And is there a way to switch to a diferent one?

**Comment by kimkulling** [comment, marked as answer]
You can find this information here: https://assimp-docs.readthedocs.io/en/latest/usage/use_the_lib.html

> By default, all 3D data is provided in a right-handed coordinate system such as OpenGL uses. In this coordinate system, +X points to the right, +Y points upwards and +Z points out of the screen towards the viewer. Several modeling packages such as 3D Studio Max use this coordinate system as well (or a rotated variant of it). By contrast, some other environments use left-handed coordinate systems, a prominent example being DirectX. If you need the imported data to be in a left-handed coordinate system, supply the #aiProcess_MakeLeftHanded flag to the ReadFile() function call.

**Reply by MichaelBelousov** [reply]
new link? https://the-asset-importer-lib-documentation.readthedocs.io/en/latest/usage/use_the_lib.html
